frontend/src/bundles/friends/pages/friends/friends.tsx

```diff
@@ -0,0 +1,196 @@
+
+    useEffect(() => {
+        const loadAllUsers = async (): Promise<void> => {
+            await dispatch(usersActions.loadAll());
+        };
+        void loadAllUsers();
+    }, [dispatch]);
+
+    useEffect(() => {
+        const loadAllFriends = async (): Promise<void> => {
+            await dispatch(friendsActions.getFriends());
+        };
+        void loadAllFriends();
+    }, [dispatch]);
+
+    return (
+        <section className="relative flex flex-col gap-5 whitespace-normal">
+            <Tabs handleTabClick={handleTabClick} activeTab={activeTab} />
+
+            <div
+                className={`flex flex-wrap items-start gap-5 ${selectedFriendId ? 'w-[calc(100%-354px)]' : 'w-full'}`}
+            >
+                {activeTab === TabsFollowers.FIND_THE_FOLLOWERS &&
+                    (users.length > 0 ? (
+                        users.map((user) => (
+                            <div key={user.id}>
+                                <FriendCard
+                                    name={user.fullName || user.email}
+                                    id={user.id}
+                                    isFollowed={false}
+                                    isActive={true}
+                                    isCardSelected={
+                                        selectedFriendId === user.id
+                                    }
+                                    selectCard={handleSelectCard}
+                                    avatarUrl={user?.avatarUrl}
+                                    onToggleFollow={handleAddFriend}
+                                />
+                            </div>
+                        ))
+                    ) : (
+                        <div>No user found to follow.</div>
+                    ))}
+
+                {activeTab === TabsFollowers.MY_FOLLOWERS &&
+                    (friends.length > 0 ? (
+                        friends.map((friend) => (
+                            <div
+                                key={friend.userId}
+                                className={'cursor-pointer'}
+                            >
+                                <FriendCard
+                                    name={friend.fullName || friend.email}
+                                    id={friend.userId}
+                                    isFollowed={true}
+                                    isActive={true}
+                                    isCardSelected={
+                                        selectedFriendId === friend.userId
+                                    }
+                                    selectCard={handleSelectCard}
+                                    avatarUrl={friend?.avatarUrl}
+                                    onToggleFollow={handleRemoveFriend}
+                                />
+                            </div>
+                        ))
+                    ) : (
+                        <div>You do not follow anyone yet.</div>
```

Comment:
You are already in friends folder and don't need to create `pages/friends`
Just make it pages/friends.tsx